src/components/manage/Form/InlineForm.jsx

```diff
@@ -137,8 +133,12 @@ const InlineForm = ({
                   {map(fieldset.fields, (field) => (
                     <Field
                       {...schema.properties[field]}
-                      id={fieldIndex ? `${field}-${fieldIndex}` : field}
-                      value={schema.properties[field].value || formData[field]}
+                      id={field}
```

Comment:
should be made unique. The problem is with fields such as "title" that are very common, will have duplicate id